### ISSUE TITLE
include with BA & BAR

### ISSUE THREAD

**Issue by eronoobos** [eronoobos/BABAR-The-Shardifant#52]
@pandaro just want to mention that i have been committing the Lua AI to the two games
http://imolarpg.dyndns.org/trac/balatest/log/

BA version 9.36 includes the Lua AI, a version just before you added your factory module. it unfortunately contains the `OwnerCreated()` bug I accidentally introduced :-(

at least it's possible for a player to use this thing

**Comment by pandaro** [comment]
Hallelujah :100: 

you do it again? I mean, since we have a lot to do, when you feel is the moment, you have the authority to update?

**Comment by eronoobos** [comment]
yes, I have commit access to the BA/BAR svn server

**Comment by pandaro** [comment]
great, im feel happy
just a suggest
if im not wrong, the name of shard is ShardLua<notVersioned> or so...
when you update it can you give it a name, or date or version id, something with it i can understand the level of update

**Comment by eronoobos** [comment]
yes, i should do that. probably i should make a hook that when i make a tag it copies stuff to the BA and BAR repositories and renames the AI accordingly

**Comment by eronoobos** [comment]
https://github.com/eronoobos/BABAR-The-Shardifant/commit/9a1fbdf796a20fec7d246f0bd5c9f3b1a9ae16cd

**Comment by eronoobos** [comment]
it is now known in BA and BAR as "BABAR 1 for ShardSpringLua 1", where 1 is the current version number of the respective parts

**Comment by eronoobos** [comment]
er, i moved "BABAR 1 for ShardSpringLua 1" to the description of the AI, because of course AILoader.lua has to have the name that LuaAI.lua specifies

**Comment by eronoobos** [comment]
btw, the version numbers in the AI description refer to https://github.com/eronoobos/BABAR-The-Shardifant/releases

i'm using flat version numbers (no periods) because differentiating between major and minor changes seems like a whole lot of work

**Comment by pandaro** [comment]
ok, i'm fallow the tag system, flat number are wonderful, now please give me feedback if i correctly understand:
1 sometime you push a tag version
2 sometime you push on BA and BAR games repo a tagged version of shard
3 inside this game we can see what tag have us shard right? clicking on the

**Comment by eronoobos** [comment]
I only push tags when I update the Lua AI bundled with BA & BAR. The tag numbers that are included with the game can be seen in the AI's description.

**Comment by eronoobos** [comment]
@pandaro i'm quite bad at playing the game, but: it beat me! http://replays.springrts.com/replay/0ebc925780dc2274b4f3e3d893560d0e/

**Comment by pandaro** [comment]
lol he face sitting on your comm!
but is not really a win, atm of your die, you produce 500 energy and shard 200 or so, was the first attack.
anycase we are making a nice work

**Comment by pandaro** [comment]
as an aside [the big bamboo eco](http://imgur.com/a/dXbPg)
with some little optimization to the economy i reach this:
55 min
40k E
750 M
lots of nanos
6 factories
3 nuke but no antinuke (hehe with this center is a little risk)
large use of t2 converters and afus
muhuhahaha (sorry)
